cli/src/app.rs

```diff
@@ -347,7 +347,49 @@ pub fn add_subcommands(command: Command) -> Command {
                             .help("Name for the new group")
                             .required(true),
                     ),
-                ),
+                )
+                .subcommand(
+                    Command::new("members").about("List group members").args(&[
+                        Arg::new("group")
+                            .short('g')
+                            .long("group")
+                            .value_name("GROUP")
+                            .help("Group to list the members for")
+                            .required(true),
+                        Arg::new("json")
+                            .action(ArgAction::SetTrue)
+                            .short('j')
+                            .long("json")
+                            .help("Produce group list in json format (default: false)"),
```

Comment:
This does not work as expected:

* `phylum group members --group x --json` outpus JSON ✅
* `phylum group members --group x list --json` outputs JSON ✅
* `phylum group members --group x --json list` outputs a table ❌
* `phylum group --json members --group x list` outputs a table ❌
* `phylum --json group members --group x list` is an error 🤷 (included for completeness)

Reply:
> I'm pretty sure this should be consistent with our other commands?

I pointed it out because I thought it was inconsistent. I was thinking of `phylum project list` which has these cases:

* `phylum project list --json` outputs JSON
* `phylum project --json list` outputs JSON

But I hadn't checked `phylum group list` which has these cases.

* `phylum group list --json` outputs JSON
* `phylum group --json list` outputs a table

So we are currently inconsistent on how we handle this... That makes this a separate issue and I will open an issue so that we can discuss a resolution.

> I personally think that our command/command list duplication is kind of a hot mess anyway.

I agree. We should also discuss plans to clean up that mess in that separate issue.